tests/platform_helper/fixtures/pipeline/platform-config-with-invalid-wildcard-branch.yml

```
@@ -1,13 +1,15 @@
+application: my-app
+legacy_project: True
```

```suggestion
legacy_project: true
```

The uppercase `T`s look odd. Elsewhere we have used `true`.